http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -568,6 +583,19 @@ public Builder tlsKeyManagersProvider(TlsKeyManagersProvider tlsKeyManagersProvi
             return this;
         }
 
+        @Override
+        public Builder initialWindowSize(Integer initialWindowSize) {
+            if (initialWindowSize != null) {
+                Validate.isPositive(initialWindowSize, "initialWindowSize");
```

Comment:
nit: we have `Validate.isPositiveOrNull` that ignores null

Reply:
+1

Reply:
There was an existing `Validate.isPositiveOrNull` but the only implementation I could find was for `Duration`. I added one for `Integer`.